specification/ai/Azure.AI.Agents/tools/models.tsp

```diff
@@ -536,6 +572,41 @@ model RunStepMcpToolCall extends RunStepToolCall {
   serverLabel?: string;
 }
 
+@doc("""
+  A record of a call to a Computer Use tool, issued by the model in evaluation of a defined tool, that represents
+  executed computer automation actions.
+  """)
+@added(Versions.v2025_05_15_preview)
+model RunStepComputerUseToolCall extends RunStepToolCall {
```

Comment:
We also require a RunStepDeltaComputerUseToolCall model and a RunStepDeltaComputerUseDetails model to be defined in run_steps/models.tspw